Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/MainActivity.cs

```diff
@@ -30,7 +30,6 @@ protected override void OnCreate(Bundle savedInstanceState)
             Xamarin.Forms.Forms.Init(this, savedInstanceState);
 
             AndroidAnalytics.SetListener(new AndroidAnalyticsListener());
-            Push.SetSenderId("177539951155");
```

Comment:
We didn't remove this in Sasquatch Android for backwards compat, we probably want to do this here, too.

Reply:
@TroubleMakerBen Sasquatch has 2 build variants, we don't have that in Xamarin.